lib/jblond/Diff/Similarity.php

```diff
@@ -223,4 +223,39 @@ private function ratioReduce(int $sum, array $triple): int
     {
         return $sum + ($triple[count($triple) - 1]);
     }
+
+    /**
+     * Get diff statistics
+     *
+     * @return array
+     */
+    public function getDifference(): array
+    {
+        $return = [
+            'inserted'     => 0,
+            'deleted'      => 0,
+            'notModified'  => 0,
```

Comment:
Array index is immediately overwritten before accessing

Reply:
I don't understand what you mean.